### PR TITLE
Minor fixes in tools

### PR DESCRIPTION
**Pull request by MarceloSalazar**
@iriark01

### CODE REVIEW

File: docs/tools/tools_intro.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ You can build Mbed OS with the Arm Compiler and GUN Arm Embedded toolchains. The
 
 | Compiler| Download location | Name in Mbed CLI |
 | --- | --- | --- |
-| Arm Compiler 6.13 (default ARM toolchain) | - A paid version is available as [Arm Compiler 6.13 Professional](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6). </br> - A paid version is also included in [Keil MDK 5.29](http://www2.keil.com/mdk5/529) | `arm` |
-| GNU Arm Embedded version 9 (9-2019-q4-major) | [GNU Arm Embedded version 9 (9-2019-q4-major)](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads) | `gcc` |
+| Arm Compiler 6.13 (default ARM toolchain) | - A paid version is available as [Arm Compiler 6.13 Professional](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6). </br> - A paid version is also included in [Keil MDK 5.29](http://www2.keil.com/mdk5/529) | `ARM` |
```
Comment (iriark01) [inline docs/tools/tools_intro.md:32]: Hang on - are they actually case sensitive?

Comment (MarceloSalazar) [reply docs/tools/tools_intro.md:32]: They're not case sensitive, however, Mbed CLI shows the compiler names in uppercase, and the rest of docs and examples also use uppercase.

File: docs/tools/tools_intro.md [hunk 2]
```diff
@@ -25,18 +25,18 @@ We created the Mbed command-line tool (Mbed CLI), a Python-based tool, specifica
...
+| GNU Arm Embedded version 9 (9-2019-q4-major) | [GNU Arm Embedded version 9 (9-2019-q4-major)](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads) | `GCC_ARM` |
 
 <span class="notes">**Note**: Mbed OS 6 does not support Arm Compiler 5, IAR or uARM.</span>
 
 ## Arm Mbed Studio
 
-Mbed Studio is our new desktop IDE and is available to anyone with an Mbed account as a public beta. You can download it for Windows and Mac, and access the documentation at [os.mbed.com/docs/mbed-studio](https://os.mbed.com/docs/mbed-studio/).
+Mbed Studio is our new desktop IDE and is available to anyone with an Mbed account as a public beta. You can download it for Windows, Linux and Mac, and access the documentation at [os.mbed.com/docs/mbed-studio](https://os.mbed.com/docs/mbed-studio/).
```
Comment (iriark01) [inline docs/tools/tools_intro.md]: @JoeTheGuitarist out of curiosity, was there a reason you didn't include Linux in your text?

Comment (JoeTheGuitarist) [reply docs/tools/tools_intro.md]: This is old text, Linux was only released a couple of months ago. You might as well remove the public beta bit as well as 1.0 is about to be released.

Comment (iriark01) [reply docs/tools/tools_intro.md]:
```suggestion
Mbed Studio is our new desktop IDE and is available to anyone with an Mbed account. You can [download it for Windows, Linux and Mac](https://os.mbed.com/docs/mbed-studio/current/installing/index.html), and access the documentation at [os.mbed.com/docs/mbed-studio](https://os.mbed.com/docs/mbed-studio/).
```

**Comment by iriark01** [comment]
@MarceloSalazar we good?

**Comment by MarceloSalazar** [comment]
Yes, everything is good - feel free to merge if you agree :)

**Pull request merged**